test/upload/cli/test_upload.py

```diff
@@ -45,7 +45,12 @@ def test_upload_with_target_filename_option(self):
         s3 = boto3.resource('s3')
         s3.Bucket(TEST_UPLOAD_BUCKET).create()
 
-        UploadCommand(Namespace(file_paths=['test/bundle/sample.json'], target_filename='FOO', quiet=True))
+        args = Namespace(
```

Comment:
Is there value in testing once with no_transfer_acceleration=True to verify if the transfer config is set up correctly?

Reply:
Done